frontend/templates/fs-base.html.jinja

```diff
@@ -0,0 +1,20 @@
+{% extends "govuk_frontend_jinja/template.html" %}
+
+
+{% block head %}
+  <link rel="stylesheet" href="{{ url_for('static', filename='apply/govuk-frontend-4.0.0.min.css' ) }}"> </link>
```

Comment:
If this is the full fs base template, it feels maybe weird to point to the apply govuk-frontend?

Can we do something bold here and just have a single base template rather than a fs-base and an apply-base? Ideally one base means more consistency, but depends what our thoughts are about how we do more vertical slices.

Reply:
Upgraded to use v4.7.0 of the frontend css and js, which is compatible with the version of the macros we already used (2.7.0)

See https://github.com/LandRegistry/govuk-frontend-jinja